### ISSUE TITLE
John the ripper MD5 test is broken

### ISSUE THREAD

**Issue by aaronjwood** [phoronix-test-suite/phoronix-test-suite#413]
Not sure if this is known already but the MD5 test for john the ripper doesn't produce any results:
```
phoronix-test-suite debug-run pts/john-the-ripper



Phoronix Test Suite v9.2.1

    Installed:     pts/john-the-ripper-1.7.0


John The Ripper 1.9.0-jumbo-1:
pts/john-the-ripper-1.7.0
Processor Test Configuration
1: MD5
2: Blowfish
3: Test All Options
** Multiple items can be selected, delimit by a comma. **
Test: 1


========================================
Phoronix Test Suite v9.2.1
System Information
========================================


  PROCESSOR:          AMD Ryzen 9 3950X 16-Core
    Core Count:       32                                       
    Extensions:       SSE 4.2 + AVX2 + AVX + RDRAND + FSGSBASE 
    Cache Size:       512 KB                                   
    Microcode:        0x6000626                                

  GRAPHICS:           SVGA3D; build: RELEASE; LLVM;
    OpenGL:           2.1 Mesa 19.2.8    
    Display Driver:   modesetting 1.20.5 
    Screen:           3440x1335          

  MOTHERBOARD:        Oracle VirtualBox v1.2
    BIOS Version:     VirtualBox                
    Chipset:          Intel 440FX 82441FX PMC   
    Audio:            Intel 82801AA AC 97 Audio 
    Network:          Intel 82540EM             

  MEMORY:             60416MB

  DISK:               107GB VBOX HDD
    File-System:      ext4                          
    Mount Options:    errors=remount-ro relatime rw 
    Disk Scheduler:   MQ-DEADLINE                   

  OPERATING SYSTEM:   Ubuntu 19.10
    Kernel:           5.3.0-29-generic (x86_64)                                                            
    Desktop:          GNOME Shell 3.34.1                                                                   
    Display Server:   X Server 1.20.5                                                                      
    Compiler:         GCC 9.2.1 20191008                                                                   
    System Layer:     Oracle VirtualBox                                                                    
    Security:         itlb_multihit: Not affected                                                          
                      + l1tf: Not affected                                                                 
                      + mds: Not affected                                                                  
                      + meltdown: Not affected                                                             
                      + spec_store_bypass: Mitigation of SSB disabled via prctl and seccomp                
                      + spectre_v1: Mitigation of usercopy/swapgs barriers and __user pointer sanitization 
                      + spectre_v2: Mitigation of Full generic retpoline STIBP: disabled RSB filling       
                      + tsx_async_abort: Not affected                                                      


========================================
John The Ripper (Run 1 of 1)
========================================


Test Run Command: cd /home/aaron/.phoronix-test-suite/installed-tests/pts/john-the-ripper-1.7.0/ && ./john-the-ripper --test=30  --format=md5crypt 2>&1

Will run 32 OpenMP threads
Benchmarking: md5crypt, crypt(3) $1$ (and variants) [MD5 256/256 AVX2 8x3]... (32xOMP) DONE
Many salts:	1337K c/s real, 42280 c/s virtual
Only one salt:	1248K c/s real, 39612 c/s virtual


Result Key: #_RESULT_#


Template Line: Raw:	#_RESULT_# c/s real, 768 c/s virtual


Result Parsing Search Key: "Raw:"


Result Line: 


No Test Results


Result Key: #_RESULT_#


Template Line: Many salts:	#_RESULT_# c/s real, 6660K c/s virtual


Result Parsing Search Key: "Many salts:"


Result Line: Many salts:	1337K c/s real, 42280 c/s virtual


No Test Results


The test run did not produce a result.


Log File At: /home/aaron/.phoronix-test-suite/installed-tests/pts/john-the-ripper-1.7.0/john-the-ripper-1.7.0-1581156030-1.log



The following tests failed to properly run:

    - pts/john-the-ripper-1.7.0: Test: MD5

```
Windows has the same issue.

**Comment by michaellarabel** [comment]
This should be fixed now by pts/john-the-ripper-1.7.1. John changed over to appending 'K' metric to the results, so the parser had to be updated. Thanks for reporting.

You may need to run `phoronix-test-suite openbenchmarking-refresh` in order to see the updated test.

**Comment by aaronjwood** [comment]
Thanks! It works perfectly now.
